packages/ts/generator-utils/src/dependencies/ImportManager.ts

```diff
@@ -138,6 +150,21 @@ export class DefaultImportManager extends StatementRecordManager<ImportDeclarati
     return this.#map.get(path)?.id;
   }
 
+  remove(path: string): void {
+    if (this.#map.has(path)) {
+      this.#map.delete(path);
+    }
+  }
+
+  find(predicate: (path: string) => boolean): [string, boolean, Identifier] | undefined {
```

Comment:
The import managers are designed the same in terms of API, so we should either add `find` method to every manager or use `identifiers` generator instead (it looks kinda broken right now, but we can fix it I guess)

Reply:
Added the find and remove methods the managers that were missing it.